src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -89,37 +120,10 @@ declare const __LOADER__IS_LAZY__: any;
     _newScriptTag.crossOrigin = 'anonymous';
 
     // Once our SDK is loaded
-    _newScriptTag.addEventListener('load', function () {
-      try {
-        _window.removeEventListener(_errorEvent, onError);
-        _window.removeEventListener(_unhandledrejectionEvent, onUnhandledRejection);
-
-        // Add loader as SDK source
-        _window.SENTRY_SDK_SOURCE = 'loader';
-
-        const SDK = _window[_namespace];
-
-        const oldInit = SDK.init;
-
-        // Configure it using provided DSN and config object
-        SDK.init = function (options) {
-          const target = _config;
-          for (const key in options) {
-            if (Object.prototype.hasOwnProperty.call(options, key)) {
-              target[key] = options[key];
-            }
-          }
-
-          setupDefaultIntegrations(target, SDK);
-          oldInit(target);
-        };
-
-        sdkLoaded(callbacks, SDK);
-      } catch (o_O) {
-        console.error(o_O);
-      }
+    _newScriptTag.addEventListener('load', onSentryScriptLoaded, {
+      once: true,
```

Comment:
Adding `once` and `passive` here which IMHO make sense (the handler can be cleaned up after being called), and in older browsers will just be ignored.